### PR TITLE
Medicine Reader completed and added

### PR DESCRIPTION
**Pull request by aritra1804**

### CODE REVIEW

**Comment by Niketkumardheeryan** [comment]
@aritra1804 make a proper readme.md file, you can refer other files or youtube...

**Comment by Niketkumardheeryan** [comment]
@pratishthaa @Rutikab12 what's your opinion on it...

**Comment by Rutikab12** [comment]
> _ No description provided. _

can you enhance the model a little bit more, data used is very less and proper description is not provided. Please refer to some other projects for the same. And make a descriptive PR with the issue number while pushing changes.

**Comment by aritra1804** [comment]
Issue #205 
Hey I have updated the readme as per requirement. Please, check. Also about the data part for the model, the required data is more than enough as I have used a pre-trained model and the accuracy is high. You can check the accuracy of the results.

### Description of project
- This project aims to read and extract medicine labels and information from medicines and pill containers using paddleocr. 
-  In simple terms, I used a pre-trained model from the paddle and initialized it according to my requirement. Once I got the results, I visualized the results according to my need and even showed the result in a graphical box to explain the visualization better.

**Comment by aritra1804** [comment]
@Niketkumardheeryan  Also add the level1,2,3 label in order for points
